src/Rector/MethodCall/WebTestCaseAssertResponseCodeRector.php

```diff
@@ -111,7 +112,10 @@ public function isGetLocationMethodCall(Expr $expr): bool
         return $this->valueResolver->isValue($firstArg->value, 'Location');
     }
 
-    private function processAssertResponseStatusCodeSame(MethodCall $methodCall): ?MethodCall
+    /**
+     * @param StaticCall|MethodCall $methodCall
+     */
+    private function processAssertResponseStatusCodeSame(Expr\CallLike $methodCall): ?Expr\CallLike
```

Comment:
using union for exact types should can be applied:

```suggestion
    private function processAssertResponseStatusCodeSame(StaticCall|MethodCall $methodCall): MethodCall|StaticCall|null
```